### DISCUSSION TITLE
无法使用专业版，7月份问过，说帮忙处理，一直没处理。

### DISCUSSION THREAD

**Discussion by heidelbergkai** [getlantern/lantern-forum-cn#819] [General]
RT。我的账号是812862967@qq.com
![微信图片_20240830095419](https://github.com/user-attachments/assets/360e05d2-85d9-4a29-9381-3567fa0791d1)

**Comment by heidelbergkai** [comment]
如果发不了这个邮箱，激活码可以发到kai.wang0313@gmail.com
如果这次还不给解决，就尽快退款。三天之内解决问题或者退款，要不我就支付宝举报了。

**Reply by MaxMKMax** [reply]
@heidelbergkai 您好. 不好意思, 美国正在公共假期. 要等他们明天晚上回来上线. 我给您提交问题然后给您补偿 1 个月时间吧. 真的非常抱歉. 会尽快给您处理!

**Reply by MaxMKMax** [reply]
@heidelbergkai 谢谢您的耐心等待, 我已经帮您提交问题了. 等他们回来就给您马上解决问题.

**Reply by heidelbergkai** [reply]
@Mike777Khoo 好的，我就再等两天。希望尽快处理。

**Reply by heidelbergkai** [reply]
@Mike777Khoo 请尽快处理，发我激活码，如果发不了812862967@qq.com，激活码可以发到kai.wang0313@gmail.com

**Reply by heidelbergkai** [reply]
@Mike777Khoo 为什么还不处理！激活码呢？

**Comment by daluzhuzi** [comment]
帮顶，我8月7号续费专业版，14号变免费版，蓝灯的问题报告啥用都没，邮件发不出。
账号邮箱kknd.jr@163.com，如果163有限制请帮转为daluzhuzi@gmail.com
![l1](https://github.com/user-attachments/assets/613ccd1a-6f3d-4e66-bdc5-9f5c68e8e0c1)

**Reply by MaxMKMax** [reply]
@daluzhuzi 在另一个帖子给您回复了. 已经把问题提交上去了
